airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -861,6 +871,56 @@ public boolean shouldRunConnectionManagerWorkflows() {
     return getEnvOrDefault(SHOULD_RUN_CONNECTION_MANAGER_WORKFLOWS, true);
   }
 
+  @Override
+  public boolean shouldHandleSyncControlPlaneTasks() {
+    final boolean result = getEnvOrDefault(SHOULD_HANDLE_SYNC_CONTROL_PLANE_TASKS, true);
+    if (result && !shouldRunSyncWorkflows()) {
+      throw new IllegalArgumentException(
+          String.format("%s cannot be true when %s is false", SHOULD_HANDLE_SYNC_CONTROL_PLANE_TASKS, SHOULD_RUN_SYNC_WORKFLOWS));
+    }
+    return result;
+  }
+
+  @Override
+  public String primarySyncDataPlaneTaskQueue() {
+    return getEnvOrDefault(PRIMARY_SYNC_DATA_PLANE_TASK_QUEUE, DEFAULT_PRIMARY_SYNC_DATA_PLANE_TASK_QUEUE);
+  }
+
+  @Override
+  public Set<String> connectionIdsForAwsDataPlane() {
+    final var connectionIds = getEnvOrDefault(CONNECTION_IDS_FOR_AWS_DATA_PLANE, "");
```

Comment:
nit: should these variable names have "aws" in them?  Presumably we will have multiple AWS data-planes in different regions one day...  Maybe the ENV should be stringified JSON like `{"AWS_EAST_1_DATA_PLANE": 1,2,3, "OTHER_DATA_PLANE", 4,5,6}`

Reply:
NVM! I see you comment in the description about how this is a MVP